docs/components/board/_index.md

```diff
@@ -151,7 +153,8 @@ Integrate `digital_interrupts` into your robot in the `attributes` of your board
 "digital_interrupts": [
   {
     "name": "<your-digital-interrupt-name>",
-    "pin": "<pin-number>"
+    "pin": "<pin-number>",
+    "type": "< basic | servo >"
```

Comment:
```suggestion
    "type": "<`basic` or `servo`>"
```
(opt) I like the idea but I think the two options also belongs in the attribute table, the | or also ppl might not automatically know what that means

Reply:
They are in the attribute table already, unless I'm misunderstanding you. I like the pipe because it's visually distinctive, but could go either way. Curious what Naomi says.

Reply:
I think the pipe is how that is commonly done to conserve space, so I'd stick with that way of doing it though I think it's also commonly without extra spaces so:

```suggestion
    "type": "<basic|servo>"
```

Reply:
Okay I'm applying this such that it's only on the Pi-specific page per Rand, and taking out the spaces.